ext/soap/soap.stub.php

```diff
@@ -8,28 +8,86 @@ function is_soap_fault(mixed $object): bool {}
 
 class SoapParam
 {
+    /** @var string */
```

Comment:
Lack of declared types is a BC consideration?

Reply:
AFAIR yes. But I think I just worked on this before we agreed that we can declare property types even for non-readonly ones. So I'll happily implement it either in a followup or in this PR directly!

Reply:
Probably good idea to do that in a separate PR, just in case.